app/scripts/templates/marketing_snippet.mustache

```diff
@@ -1,7 +1,7 @@
   {{#showSignUpMarketing}}
     <aside class="marketing desktop image-snippet default">
       <div class="marketing-spacer">
-        <span class="android-logo" />
+        <span class="android-logo"></span>
```

Comment:
```
Error: Self-closing syntax (/>) used on a non-void HTML element. Ignoring the slash and treating as a start tag.
From line 9, column 10; to line 9, column 38
↩         <span class="android-logo" />↩     
Error: End tag div seen, but there were open elements.
From line 13, column 8; to line 13, column 13
a>↩       </div>↩↩</bo
Error: Unclosed element span.
From line 9, column 10; to line 9, column 38
↩         <span class="android-logo" />↩     
```

Apparently `<span />` is invalid markup??! Used the slightly more verbose and annoying `<span></span>`.